commit 17589e37dff9745d9c219973a77031905cb57973
Author: Andrew Litt <ajlitt@splunge.net>
Date:   Mon Jan 11 07:54:21 2016 +0000

    bcm2835-mmc: Don't override bus width capabilities from devicetree
    
    Take out the force setting of the MMC_CAP_4_BIT_DATA host capability
    so that the result read from devicetree via mmc_of_parse() is
    preserved.

diff --git a/drivers/mmc/host/bcm2835-mmc.c b/drivers/mmc/host/bcm2835-mmc.c
index 43aed6e..104f93e 100644
--- a/drivers/mmc/host/bcm2835-mmc.c
+++ b/drivers/mmc/host/bcm2835-mmc.c
@@ -1305,7 +1305,7 @@ static int bcm2835_mmc_add_host(struct bcm2835_host *host)
 	/* host controller capabilities */
 	mmc->caps |= MMC_CAP_CMD23 | MMC_CAP_ERASE | MMC_CAP_NEEDS_POLL |
 		MMC_CAP_SDIO_IRQ | MMC_CAP_SD_HIGHSPEED |
-		MMC_CAP_MMC_HIGHSPEED | MMC_CAP_4_BIT_DATA;
+		MMC_CAP_MMC_HIGHSPEED;
 
 	mmc->caps2 |= MMC_CAP2_SDIO_IRQ_NOTHREAD;
 
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1123_17589e37dff9745d9c219973a77031905cb57973.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1123_17589e37dff9745d9c219973a77031905cb57973.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
